packages/components/combobox/src/ComboboxSelectedItems.tsx

```diff
@@ -25,72 +22,78 @@ export const SelectedItems = () => {
     }
   }
 
+  const { disabled, ...selectedItemProps } = ctx.getSelectedItemProps({
+    disabled: ctx.disabled || ctx.readOnly,
+    selectedItem,
+    index,
+  })
+
+  const Element = disabled ? 'button' : 'span'
+
   return (
-    <>
-      {ctx.selectedItems.map((selectedItemForRender, index) => {
-        const { disabled, ...selectedItemProps } = ctx.getSelectedItemProps({
-          disabled: ctx.disabled || ctx.readOnly,
-          selectedItem: selectedItemForRender,
-          index,
-        })
+    <Element
+      role="presentation"
+      data-spark-component="combobox-selected-item"
+      key={`selected-item-${index}`}
+      className={cx(
+        'flex h-sz-28 items-center rounded-md bg-neutral-container align-middle',
+        'text-body-2 text-on-neutral-container',
+        'disabled:cursor-not-allowed disabled:opacity-dim-3',
+        'outline-none focus-visible:u-ring-inset',
```

Comment:
Only this line has changed, the rest has moved because I extracted the logic inside `selectedItems` loop into a `SelectedItem` component (for clarity)